cscs-checks/system/openstack/src/s3_create_bucket.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python
```

Comment:
The correct way to do that  is the following:

```suggestion
#!/usr/bin/env python3
```

Check [here](https://stackoverflow.com/questions/6908143/should-i-put-shebang-in-python-scripts-and-what-form-should-it-take) for more details.